spec/main-spec.js

```diff
@@ -0,0 +1,37 @@
+'use babel';
+
+describe('TestingForAva', () => {
```

Comment:
Yeah, is more consistent and makes a lot of sense use AVA. I have been thinking about that since the beginning. My only doubt is if it can lead to any problems, just because Jasmine in the de facto option in Atom. 

Probably is time to make some tests.